forms-flow-api/src/formsflow_api/models/form_process_mapper.py

```diff
@@ -29,7 +29,7 @@ class FormProcessMapper(
 
     id = db.Column(db.Integer, primary_key=True)
     form_id = db.Column(db.String(50), nullable=False)
-    form_name = db.Column(db.String(100), nullable=False)
+    form_name = db.Column(db.String(200), nullable=False)
```

Comment:
is it possible to give 200 as value for a const variable and use it in other files also by importing the vvalue ? @shuhaib-aot

Reply:
it's okay to pass like this because we are using this length only for this purpose

Reply:
ok for now